src/bridge/client/cli/client_command.rs

```diff
@@ -73,32 +73,8 @@ impl ClientCommand {
         loop {
             self.client.sync().await;
 
-            let peg_in_graphs = self.client.get_data().peg_in_graphs.clone();
 
-            for peg_in_graph in peg_in_graphs.iter() {
```

Comment:
`handle_automatic_command` fn should run all actions depending on client's role (one client can have more than one role). 

We're currently missing handling verifier's commands. However we should keep depositor's operations here as we'll complete the function with verifier's commands soon.

Reply:
Right, this is a little bit of a confusing point. Our client:

• supports as roles: depositor, operator, verifier, withdrawer
• functions are implemented piece-wise per role; and a client can startup the client with multiple roles running concurrently

Reply:
Woops, my bad. I forgot to call `process_peg_ins` here - fixed now.

I also extracted the pegout processing logic into a `process_peg_outs` function which I placed into the client file as well. I think it makes more sense to have most of the logic in the client file, and have `client_command` just be an entry point into the program. If you disagree just lmk and I'll refactor it back

Reply:
I'm fine with this refactor.